SportHub/src/modules/VerticalAdminMenu/components/VerticalAdminMenu.js

```diff
@@ -19,10 +20,16 @@ export default function VerticalAdminMenu(){
                         <VerticalAdminMenuElement
                             name={verticalMenuElement}
                             hintText={hintText[index]}
-                            isActive={index === currentVerticalMenuElement}  />
+                            isActive={index === currentVerticalMenuElement} />
                     </div>
                 })}
             </div>
+            {currentVerticalMenuElement === 5 && (
+                <div className={styles.module_content_container}>
```

Comment:
Ask Roman how he is implementing this selection, or we can discuss it on Thursday